src/app/app.js

```diff
@@ -22,7 +22,13 @@
     'ui.codemirror',
     'hljs',
     'ngSanitize'
-  ]).config(['hljsServiceProvider', function (hljsServiceProvider) {
+  ]).config(['$locationProvider', 'hljsServiceProvider', function ($locationProvider, hljsServiceProvider) {
+
+    $locationProvider.html5Mode({
+      enabled: true,
+      requireBase: false
+    });
```

Comment:
@carowright not really, i used html5mode to use $location service to parse uri parameters, before it was done manually.

I'll remove it, thank you.